perf/sawtooth_perf/src/batch_submit.rs

```diff
@@ -181,17 +185,19 @@ pub fn run_workload<'a>(batch_list_iter: &'a mut Iterator<Item = BatchListResult
         .map(move |_| -> Result<BatchList, workload::WorkloadError> {
             workload::get_next_batchlist(batch_list_iter)
         })
-        .map(move |batch_list: Result<BatchList, workload::WorkloadError>| {
+        .map(|batch_list: Result<BatchList, workload::WorkloadError>| {
+            let basic_auth_c = basic_auth.clone();
```

Comment:
This is a strange naming convention.  We should talk as a team about what we want for these.